### PR TITLE
docs: switch from code gen to cli

### PR DESCRIPTION
**Pull request by beeme1mr**
Generalize the project from code gen to CLI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,28 +21,37 @@
...
+> Feel free to give it a shot and provide feedback, but expect breaking changes.
 
 [OpenFeature](https://openfeature.dev) is an open specification that provides a vendor-agnostic, community-driven API for feature flagging that works with your favorite feature flag management tool or in-house solution.
 <!-- x-hide-in-docs-end -->
 
-## Why CodeGen?
+## Overview
 
-CodeGen automates the creation of strongly typed flag accessors, minimizing configuration errors and provides a better developer experience.
+The OpenFeature CLI is a command-line tool designed to improve the developer experience of feature flags.
```
Comment (beeme1mr) [inline README.md:32]:
```suggestion
The OpenFeature CLI is a command-line tool designed to improve the developer experience when working with feature flags.
```

File: README.md [hunk 2]
```diff
@@ -21,28 +21,37 @@
...
 
 [OpenFeature](https://openfeature.dev) is an open specification that provides a vendor-agnostic, community-driven API for feature flagging that works with your favorite feature flag management tool or in-house solution.
 <!-- x-hide-in-docs-end -->
 
-## Why CodeGen?
+## Overview
 
-CodeGen automates the creation of strongly typed flag accessors, minimizing configuration errors and provides a better developer experience.
+The OpenFeature CLI is a command-line tool designed to improve the developer experience when working with feature flags.
+The initial initial version of the CLI focuses on code generation.
```
Comment (toddbaert) [inline README.md:33]: Currently, features are focused primarily on supporting code generation.

Comment (toddbaert) [reply README.md:33]: Might read better and a bit less brittle if we add minor other stuff

Comment (beeme1mr) [inline README.md:33]:
```suggestion
Currently, features are focused primarily on supporting code generation.
```

**Review by toddbaert — APPROVED** [review]
Minor recommendation

**Pull request merged**